spec/yaml/swiftnav/sbp/solution_meta.yaml

```diff
@@ -0,0 +1,222 @@
+                   - 0: Reference epoch is start of current GPS week
+                   - 1: Reference epoch is time of system startup
+                   - 2: Reference epoch is unknown
+                   - 3: Reference epoch is last PPS
+             - 6-7:
+                 desc: Reserved
+
+  # for Odometry sensors
+ - OdoInputType:
+     id: 0xFFE9
+     short_desc: Flags for a given Odometry sensor used as input for the fuzed solution.
+     public: true
+     desc: |
+       Metadata around the Odometry sensors involved in the fuzed solution.
+       Note: Just to build descriptive tables in documentation and not actually used.
+     fields:
+       - flags:
+           type: u8
+           desc: flags that store all relevant info specific to this sensor type.
+           fields:
+             - 0-1:
+                 desc: Odometer class
+                 values:
+                   - 0: single or averaged ticks
+                   - 1: single or averaged speed
+                   - 2: multi-dimensional ticks
+                   - 3: multi-dimensional speed
+             - 2-3:
+                 desc: Odometer grade
+                 values:
+                   - 0: Low Grade (e.g. quantisized CAN)
+                   - 1: Medium Grade
+                   - 2: Superior Grade
+                   - 3: Reserved
+             - 4-5:
+                 desc: Rate
+                 values:
+                   - 0: Fixed incoming rate
+                   - 1: Incoming when triggered by minimum distance or speed
+                   - 2: Reserved
+                   - 3: Reserved
+             - 6-7:
+                 desc: Reserved
+
+  # the actual type of the elements going into the array solution_inputs in MSG_SOLN_META
+ - SolutionInputType:
+     short_desc: Flags for a given solution input type.
+     desc: |
+       Metadata describing which sensors were involved in the solution.
+       The structure is fixed no matter what the actual sensor type is.
+       The sensor_type field tells you which sensor we are talking about. It also tells you
+       whether the sensor data was actually used or not.
+       The flags field, always a u8, contains the sensor-specific data.
+       The content of flags, for each sensor type, is described in the relevant structures in this section.
+     type: u16
+     fields:
+       - sensor_type:
+           type: u8
+           desc: The type of sensor
+           fields:
+             - 0-2:
+                 desc: The type of sensor
+                 values:
+                   - 0: GNSS Position (see GNSSInputType)
+                   - 1: GNSS Velocity Displacement (see GNSSInputType)
+                   - 2: GNSS Velocity Doppler (see GNSSInputType)
+                   - 3: Odometry Ticks (see OdoInputType)
+                   - 4: Odometry Speed (see OdoInputType)
+                   - 5: IMU Sensor (see IMUInputType)
+                   - 6: Reserved
+                   - 7: Reserved
```

Comment:
2 more sensors seems a little tight here, Im wondering if we should rather to 0-3 for type of sensor, 4-5 for description and 6-7 for reserved

Reply:
Good point, Done!